src/main/java/core/basesyntax/temp/FigureSupplier.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.temp;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    static final String DEFAULT_COLOR = "WHITE";
+    static final double DEFAULT_RADIUS = 10.0;
+    static final int FIGURE_AMOUNT = 4;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Figure figure = new Circle();
+
+        int figureNumber = random.nextInt(FIGURE_AMOUNT);
+
+        switch (figureNumber) {
+            case (0):
+                figure = getRandomCircle();
+                break;
+            case (1):
+                figure = getRandomIsoscelesTrapezoid();
+                break;
+            case (2):
+                figure = getRandomRectangle();
+                break;
+            case (3):
+                figure = getRandomRightTriangle();
+                break;
+            case (4):
+                figure = getRandomSquare();
+                break;
+            default:
+                System.out.println("Wrong random number");
+        }
+        return figure;
+    }
+
+    public Figure getRandomCircle() {
+        Circle circle = new Circle();
+        circle.setRadius(random.nextDouble());
+        circle.setColor(colorSupplier.getRandomColor());
+        circle.getArea();
+        Figure figure = circle;
+        return figure;
```

Comment:
```suggestion
        return circle
```
the same for methods below